.travis.yml

```diff
@@ -41,4 +41,4 @@ notifications:
 matrix:
   fast_finish: true
   allow_failures:
-  - go: tip
```

Comment:
I believe this is changed back to original, @stack72 - let me know if it's still different